tools/beerocks_analyzer/beerocks_analyzer.py

```diff
@@ -13,46 +13,54 @@
 
 bra = None
 app = None
 
 logger_setup.setup_logger()
 logger = logging.getLogger(__name__)
 
+
 def printUsage():
     logger.info("usage: BeeRocksAnalyzer [options]")
-    logger.info("    -f=<input log file>                                         - input log file")
-    logger.info("    -map                                                  - start connectivity map")
-    logger.info("    -map_win                                              - start connectivity map on a separare window")
-    logger.info("    -conf=<conf_file_name>                                - run configuration from ./conf/conf_file_name")
-    logger.info("                                                          - can execute with several -conf=<conf_file_name1> -conf=<conf_file_name2> ...")
-    logger.info("    -gw_ip=<IP of GW>                                     - GW ip to get updates from. If empty, the GW is assumed to be running locally")
-    logger.info("    -bin_path=<path_to_beerocks_cli>                      - Path to the beerocks_cli binary. If empty, use build from <top_level>/../")
-    logger.info("    -my_ip=[IP of PC]                                     - PC ip to send updates, if not set will automaticly set")
-    logger.info("    -ssh_port=[ssh port of GW]                            - SSH port used to connect to GW")
-    logger.info("    -docker_container=[name of container]                 - Name of the docker container on which to run beerocks_cli")
+    logger.info("    -f=<input log file>                                   "
+                "- input log file")
+    logger.info("    -map                                                  "
+                "- start connectivity map")
+    logger.info("    -map_win                                              "
+                "- start connectivity map on a separare window")
+    logger.info("    -conf=<conf_file_name>                                "
+                "- run configuration from ./conf/conf_file_name")
+    logger.info("                                                          "
+                "- can execute with several -conf=<conf_file_name1> -conf=<conf_file_name2> ...")
+    logger.info("    -gw_ip=<IP of GW>                                     "
+                "- GW ip to get updates from. If empty, the GW is assumed to be running locally")
+    logger.info("    -bin_path=<path_to_beerocks_cli>                      "
+                "- Path to the beerocks_cli binary. If empty, use build from <top_level>/../")
+    logger.info("    -my_ip=[IP of PC]                                     "
+                "- PC ip to send updates, if not set will automaticly set")
+    logger.info("    -ssh_port=[ssh port of GW]                            "
+                "- SSH port used to connect to GW")
+    logger.info("    -docker_container=[name of container]                 "
+                "- Name of the docker container on which to run beerocks_cli")
```

Comment:
What a mess... This shouldn't even be here, it should be handled by a library like argparse.

For now if you have the opportunity, could you just remove all the extra spaces before the second dash? So the first one would be like this for example:
```
"    -f=<input log file> - input log file"
```
It should make it look a tiny bit less messy.